internal/packages/internal/packagedeploy/chunking.go

```diff
@@ -44,8 +64,12 @@ func determineChunkingStrategyForPackage(pkg adapters.GenericPackageAccessor) ob
 	switch chunkingStrategy(strategy) {
 	case chunkingStrategyEachObject:
 		return &EachObjectChunker{}
-	default:
+	case chunkingStrategyBinpackNextFit:
+		return &BinpackNextFitChunker{}
```

Comment:
nit: isn't this redundant given that the `default` branch also returns `&BinpackNextFitChunker{}`?

Reply:
I find it actually good for understanding, the case is explicitly considered and by coincidence does the same as "anything else"

Reply:
I'll keep it :)